libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.java

```diff
@@ -80,13 +80,15 @@
 
     // Config
     private BootConfig bootconfig;
+    private AuthConfigUtil.MyDomainAuthConfig authConfig;
 
     // Web app loaded?
     private boolean webAppLoaded = false;
 
     public SalesforceDroidGapActivity() {
         super();
         delegate = new SalesforceActivityDelegate(this);
+        authConfig = AuthConfigUtil.getMyDomainAuthConfig(SalesforceHybridSDKManager.getInstance().getLoginServerManager().getSelectedLoginServer().url);
```

Comment:
Fetching it here so that it'll do it only once (when the hybrid app is loaded).

Reply:
What if login is deferred, wouldn't that code run too soon?

Reply:
It doesn't matter. In both cases, it runs only once in the lifecycle of this view controller. Also, the endpoint is open to all, so it has nothing to do with auth state of the app. Even if we deferred this call to after login, it would still run only once.

Reply:
What if you select a different login server and are using deferred login, wouldn't you be using the wrong auth config then?

Reply:
Selecting a different login server in the container would lead to this component being recreated. The selected login server should correspond with the selected one at creation time of this VC in a 1:1 manner always.

Reply:
Is that true for an hybrid local app?
Doesn't the app continue after deferred login without its state being reset?

Reply:
It's true for both apps because the only way through UI to switch servers today is through the login screen and when we tear down the login screen, we reload this activity.